src/test/scala/com/typesafe/jse/TriremeSpec.scala

```diff
@@ -75,7 +75,7 @@ class TriremeSpec extends Specification {
         .count(_.getName.contains("Trireme"))
 
       triremeThreadCount must_== 0
-      ok
+      skipped("Skipping as it always fails on TravisCI")
```

Comment:
Definitely do not skip the Trireme thread leak test - if it's failing, it most likely means the Trireme upgrade has introduced a thread leak, in which case, that needs to be fixed, not ignored.